packages/db/tests/models/test-async-operation-model.js

```diff
@@ -0,0 +1,75 @@
+const test = require('ava');
+const cryptoRandomString = require('crypto-random-string');
+
+const {
+  AsyncOperationPgModel,
+  generateLocalTestDb,
+  destroyLocalTestDb,
+} = require('../../dist');
+
+const { fakeAsyncOperationRecordFactory } = require('../../dist/test-utils');
+
+const { migrationDir } = require('../../../../lambdas/db-migration');
+
+const testDbName = `rule_${cryptoRandomString({ length: 10 })}`;
+
+test.before(async (t) => {
+  const { knexAdmin, knex } = await generateLocalTestDb(
+    testDbName,
+    migrationDir
+  );
+  t.context.knexAdmin = knexAdmin;
+  t.context.knex = knex;
+
+  t.context.asyncOperationPgModel = new AsyncOperationPgModel();
+});
+
+test.beforeEach((t) => {
+  t.context.asyncOperationRecord = fakeAsyncOperationRecordFactory();
+});
+
+test.after.always(async (t) => {
+  await destroyLocalTestDb({
+    ...t.context,
+    testDbName,
+  });
+});
+
+test('AsyncOperationPgModel.upsert() creates new async operation', async (t) => {
+  const {
+    knex,
+    asyncOperationPgModel,
+    asyncOperationRecord,
+  } = t.context;
+
+  await asyncOperationPgModel.upsert(knex, asyncOperationRecord);
+
+  t.like(
+    await asyncOperationPgModel.get(knex, asyncOperationRecord),
+    asyncOperationRecord
+  );
+});
+
+test('AsyncOperationPgModel.upsert() overwrites an async operation record', async (t) => {
+  const {
+    knex,
+    asyncOperationPgModel,
+    asyncOperationRecord,
+  } = t.context;
+
+  await asyncOperationPgModel.create(knex, asyncOperationRecord);
+
+  const updatedRule = {
```

Comment:
Change `updatedRule` to `updatedAsyncOperation`

Reply:
Gross. This was (obviously) copy pasta.

https://github.com/nasa/cumulus/pull/2162/commits/ac551f1e053ed9dccc8b009a8fa0e0ab0aab177c